__posts/2022-06-02-release-2.2.0.md

```diff
@@ -87,7 +87,7 @@ For a full list of bugfixes see the [CHANGELOG][].
 
```

Comment:
30 contributors, 21 of which are new contributors

New contributors are:
BFedder, 
inomag, 
Agorfa, 
aya9aladdin, 
shudipto-amin
HenokB,
umak1106,
tamandeeps,
Mrqeoqqt,
megosato,
AnirG,
rishu235,
manishsaini6421,
Sukeerti1,
robotjellyzone,
markvrma,
alescoulie,
mjtadema,
aditi2906,
rsexton2,
rafaelpap,

Reply:
thanks for the stats, added